### DISCUSSION TITLE
unexpected RPC response ID from Go plugin server

### DISCUSSION THREAD

**Discussion by liorjosef-guesty** [Kong/kong#11774] [Help]
I'm using kong go plugin which perform JWT authentication via okta.
it was working perfectly for more than a year and suddenly from time to time the kong instance stop to work properly (returns 500) with the following error:
"unexpected RPC response ID from Go plugin server"

I found also the source of this here: https://github.com/Kong/kong/blob/f0427c9a120621505ca8a6d87bf81807d4ace36d/kong/runloop/plugin_servers/mp_rpc.lua#L237

Any idea what can be the reason? a direction to investigate ?

Regards,

**Comment by liorsep** [comment]
FYI, Kong version is 2.7.0
